docker/README.md

```diff
@@ -78,7 +79,8 @@ The volume mounted to `/opengrok/src` should contain the projects you want to ma
 `INDEXER_OPT` | empty | pass **extra** options to OpenGrok Indexer. The default set of indexer options is: `--remote on -P -H -W`. For example, `-i d:vendor` will remove all the `*/vendor/*` files from the index. You can check the indexer options on https://github.com/oracle/opengrok/wiki/Python-scripts-transition-guide
 `NOMIRROR` | empty | To avoid the mirroring step, set the variable to non-empty value.
 `URL_ROOT` | `/` | Override the sub-URL that OpenGrok should run on.
-`WORKERS` | number of CPUs in the container | number of workers to use for syncing
+`WORKERS` | number of CPUs in the container | number of workers to use for syncing (applies only to setup with projects enabled)
+`AVOID_PROJECTS` | False | run in project less configuration. Set to non empty value disables projects.
```

Comment:
Should not `empty` be used instead of `False`?

Reply:
ok